clients/src/main/java/org/apache/kafka/clients/ClientUtils.java

```diff
@@ -46,11 +46,11 @@
                 try {
                     InetSocketAddress address = new InetSocketAddress(host, port);
                     if (address.isUnresolved()) {
-                        log.warn("Removing server from " + CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG + " as DNS resolution failed: " + url);
+                        log.warn("Removing server from " + CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG + " as DNS resolution failed: " + host);
                     } else {
                         addresses.add(address);
                     }
-                } catch (NumberFormatException e) {
+                } catch (IllegalArgumentException e) {
```

Comment:
Since NumberFormatException is a descendent of IllegalArgumentException shouldn't this catch both?

https://docs.oracle.com/javase/7/docs/api/java/lang/NumberFormatException.html